server/util/ipfs.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import axios from 'axios';
+const FormData = require('form-data');
+import models from '../database';
+require('dotenv').config();
+
+const ipfs = async (userID, addressID, jsonfile) => {
+  const data = new FormData();
+  data.append('file', JSON.stringify(jsonfile), 'userIDblob');
+  const headers = {
+    pinata_api_key: process.env.PINATA_API_KEY,
```

Comment:
We should explicitly check that `PINATA_API_KEY` exists and fail if it doesn't exist, before making the call.

Reply:
Done